Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -208,6 +206,21 @@ void GameFile::SaveCache()
   // TODO
 }
 
+QString GameFile::GetFileName() const
+{
+  return QFileInfo(m_path).fileName();
```

Comment:
Why a separate commit when it coincides directly with what the first commit does? It (IMO) doesn't really make sense to specify a dependency after you've used it.